deploy/systemd/dht@.service

```diff
@@ -0,0 +1,48 @@
+# Runs a dhtnode instance
+
+[Unit]
+Description=Distributed Hash Table instance %I
+AssertPathExists=/srv/dhtnode/dhtnode-%i
+Requires=network-online.target local-fs.target
+After=network-online.target local-fs.target
+#StartLimitIntervalSec=
+#StartLimitBurts=
+
+[Service]
+Type=simple
+WorkingDirectory=/srv/dhtnode/dhtnode-%i/
+ExecStart=/usr/sbin/dhtnode -c /srv/dhtnode/dhtnode-%i/etc/config.ini
+ExecReload=/bin/kill -HUP $MAINPID
+
+# Not sure about this one
+#ExecReload=/bin/kill -HUP $MAINPID
+
+# SIGKILL only manually
+TimeoutStopSec=infinity
+SendSIGKILL=no
+Restart=on-failure
+User=dhtnode
+Group=core
```

Comment:
Note that your package is not creating the user / group.

Reply:
How's that? 

```
if [ "$1" = "configure" ]; then
    addgroup --system core
    adduser --system --no-create-home dhtnode

```